src/main/java/spark/SparkInstance.java

```diff
@@ -327,11 +327,11 @@ public synchronized void init() {
      * @param exceptionClass the exception class
      * @param handler        The handler
      */
-    public synchronized void exception(Class<? extends Exception> exceptionClass, ExceptionHandler handler) {
+    public synchronized <T extends Exception> void exception(Class<T> exceptionClass, ExceptionHandler<? super T> handler) {
```

Comment:
The `super` allows for a more general `ExceptionHandler` to be used for a more specific `Exception`. Example:

```java
static class MoreSpecificException extends RuntimeException {}

static class RuntimeExceptionHandler implements ExceptionHandler<RuntimeException> {
    @Override
    public void handle(RuntimeException exception, Request request, Response response) {
    }
}

@Test
public void testName() throws Exception {
     Spark.exception(MoreSpecificException.class, new RuntimeExceptionHandler());
}
```